libexec/scoop-uninstall.ps1

```diff
@@ -54,6 +54,8 @@ if (!$apps) { exit 0 }
         $dir = versiondir $app $version $global
         $persist_dir = persistdir $app $global
 
+        Invoke-HookScript -HookType 'pre_uninstall' -Manifest $manifest -Arch $architecture
```

Comment:
Move variable assignments `$manifest` `$install` `$architecture` below to top otherwise they will be null which is not expected.